packages/backend/src/app.ts

```diff
@@ -345,7 +347,8 @@ export class App {
     }
     router.get(
       '/keys/{keyId}',
```

Comment:
Shouldn't this route have been removed?

Reply:
I'm guessing it will with:
- https://github.com/interledger/rafiki/pull/678